sdk/include/reactos/libs/libxml/xmlversion.h

```diff
@@ -171,7 +171,7 @@ XMLPUBFUN void XMLCALL xmlCheckVersion(int version);
  *
  * Whether the FTP support is configured in
  */
-#if 1
+#if 0
 #define LIBXML_FTP_ENABLED
```

Comment:
This one makes sense to have disabled, but it was enabled before.
Did you disable this intentionally?

Reply:
Version 2.10.0 changed the default from 1 to 0, so I didn't do it explicitly, it just happened.